### PR TITLE
Deprecate iOS 10

### PR DESCRIPTION
**Pull request by kyleve**
See ya!

Also moves us to test on 11/12/13/14

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,7 @@ git:
 
 language: swift
 os: osx
-osx_image: xcode11.5
-
-# https://docs.travis-ci.com/user/caching/
-cache:
```
Comment (kyleve) [inline .travis.yml:9]: Removing this since I don't think it does a ton now with the Cocoapods CDN... verifying.

Comment (kyleve) [reply .travis.yml:9]: Ok I think it made it slower, bringing it back

Comment (kyleve) [reply .travis.yml:9]: It is back

**Review by kylebshr — APPROVED** [review]
Hmm I really like how all the tests are named with this set up - that's one downside of just using a matrix like I did with BP

**Comment by kyleve** [comment]
Tbh, one reason I did it this way since i couldn't figure out how to get travis' matrices working. Other reason I did it this way is so that you can locally run the `run_all_tests.sh` script to run the tests across all iOS versions easily; useful for snapshot tests.

**Comment by kylebshr** [comment]
Super useful. Maybe we can steal it some day

**Pull request merged**